RadixWallet/Clients/CacheClient/CacheClient+Live.swift

```diff
@@ -61,7 +61,7 @@ extension CacheClient: DependencyKey {
 			@Dependency(\.diskPersistenceClient) var diskPersistenceClient
 
 			do {
-				try diskPersistenceClient.removeAll()
+				try diskPersistenceClient.remove(Entry.root)
```

Comment:
Should we delete the `removeAll` from `DiskPersistenceClient`?

Reply:
I did choose to still have it, in case we do really need to clear the whole `caches` directory.